test/Log4NetAppender.Net45.Tests/ApplicationInsightsAppenderTests.cs

```diff
@@ -162,20 +162,7 @@ public void TelemetryIsAcceptedByValidateEndpoint()
 
             Assert.IsNull(TelemetrySender.ValidateEndpointSend(telemetry));
         }
-
-        [TestMethod]
-        [TestCategory("Log4NetAppender")]
-        public void Log4NetSetsAuthenticatedUser()
```

Comment:
Would you mind changing this test into Log4NetDoesNotSetAuthenticatedUser() rather than remove it?
With this we will enforce this new requirement we've got.

Reply:
Makes sense. Done.